pyramid_oereb/lib/config.py

```diff
@@ -86,12 +89,15 @@ def get_theme(code):
 
         plrs = Config._config.get('plrs')
         if plrs and isinstance(plrs, list):
+            position = 1
             for theme in plrs:
```

Comment:
Because plrs is a list, you can just use [enumerate](http://book.pythontips.com/en/latest/enumerate.html) and get rid of the position:

`for position, theme in enumerate(plrs, start=1):`

The start property allows the ongodly choice of starting to count with 1...

Reply:
Thanks

Reply:
There are two other instances in your PR where you can do the same.